src/replays/_replayswidget.py

```diff
@@ -722,13 +779,23 @@ def spoilerCheckboxchange(self, state):
                 self.selectedReplay.generateInfoPlayersHtml()  # then we redo it
 
     def resetRefreshPressed(self):  # reset search parameter and reload recent Replays List
-        self._w.searchInfoLabel.setText(self.searchInfo)
-        self.vault_connection.connect_()
-        self.vault_connection.send(dict(command="list"))
-        self._w.minRating.setValue(0)
-        self._w.mapName.setText("")
-        self._w.playerName.setText("")
-        self._w.modList.setCurrentIndex(0)  # "All"
+        if not self.searching:
+            self._w.searchInfoLabel.setText(self.searchInfo)
+            self.searching = True
+            
+            parameters = {  "page[number]": 1,
+                            "page[size]": 50,
+                            "sort": "-startTime",
+                            "endTime": "isnull=false",
+                            "include": "featuredMod,mapVersion,mapVersion.map,playerStats,playerStats.player"}
```

Comment:
What i see is tripple time copy-paste kung-fu, exctract `parameters` to to constant

Reply:
true :)